src/Scripting/Core/ScriptOptions.cs

```diff
@@ -54,6 +53,7 @@ private static ImmutableArray<MetadataReference> GetDefaultMetadataReferences()
                 "System.Reflection.Extensions",
                 "System.Reflection.Primitives",
                 "System.Runtime",
+                "System.Runtime.Extensions",
```

Comment:
Are any tests failing without it? Note that scripting references are recursive, so if Microsoft.CSharp.dll references System.Runtime.Extensions it will be automatically referenced.